### DISCUSSION TITLE
Not seeing all source code in Src folder such as Components, Elements, Styles (New to Gatsby themes)

### DISCUSSION THREAD

**Discussion by kirill-develops** [LekoArts/gatsby-themes#972] [Questions] [answered]
Hi there,

I noticed when I installed the theme, that none of the src folder was populated with the exception of the sections folder and the MDX files within it. Wondering if I'm going about it wrong as I see in the theme folder on the repo there is a few other folders that I cannot access (components, elements, hooks, etc)

**Comment by LekoArts** [comment, marked as answer]
Hi!

If they don't exist, create the files yourself. The files you see exist as an example to show how things are done. If you want to change other files, create files to override things. That's the purpose of a theme, that unless you want to change something you just consume it.

**Comment by DarkhatRS** [comment]
> Hi there,
> 
> I noticed when I installed the theme, that none of the src folder was populated with the exception of the sections folder and the MDX files within it. Wondering if I'm going about it wrong as I see in the theme folder on the repo there is a few other folders that I cannot access (components, elements, hooks, etc)

Hi, how did you solve it?

**Reply by nickburczyk** [reply]
@LekoArts I have a the same issue. I think what I am struggling with is seeing how the site is built at all. I'd like to (for example) add a page section for each example in my portfolio (maximum 3). Or maybe a modal to open when someone clicks on "project card" but I don't see where those components are or how they are being utilized in order to create and consume components of my own. 

For example, where is the "index.tsx" file that serves as the layout or homepage? Or where is the page component with the query that pulls in the mdx files? Any more in-depth details you could offer would be helpful.

@kirill-develops Did you solve this? What did you do?
